### ISSUE TITLE
building a wheel fails due to LICENSE not in source package

### ISSUE THREAD

**Issue by miguelgrinberg** [rackerlabs/yoke#39]
Example:

```
$ pip wheel --no-binary :all: -w . yoke
Collecting yoke
  Using cached yoke-0.3.0.tar.gz
Collecting boto3>=1.4.2 (from yoke)
  Using cached boto3-1.4.4.tar.gz
Collecting botocore>=1.4.85 (from yoke)
  Using cached botocore-1.5.45.tar.gz
Collecting Jinja2>=2.8 (from yoke)
  Using cached Jinja2-2.9.6.tar.gz
Collecting jsonref>=0.1 (from yoke)
  Using cached jsonref-0.1.tar.gz
Collecting lambda-uploader>=1.1.0 (from yoke)
  Using cached lambda-uploader-1.1.0.tar.gz
Collecting retrying>=1.3.3 (from yoke)
  Using cached retrying-1.3.3.tar.gz
Collecting ruamel.yaml>=0.11.11 (from yoke)
  Using cached ruamel.yaml-0.14.11.tar.gz
Collecting jmespath<1.0.0,>=0.7.1 (from boto3>=1.4.2->yoke)
  Using cached jmespath-0.9.2.tar.gz
Collecting s3transfer<0.2.0,>=0.1.10 (from boto3>=1.4.2->yoke)
  Using cached s3transfer-0.1.10.tar.gz
Collecting python-dateutil<3.0.0,>=2.1 (from botocore>=1.4.85->yoke)
  Using cached python-dateutil-2.6.0.tar.gz
Collecting docutils>=0.10 (from botocore>=1.4.85->yoke)
  Using cached docutils-0.13.1.tar.gz
Collecting MarkupSafe>=0.23 (from Jinja2>=2.8->yoke)
  Using cached MarkupSafe-1.0.tar.gz
Collecting virtualenv (from lambda-uploader>=1.1.0->yoke)
  Using cached virtualenv-15.1.0.tar.gz
Collecting six>=1.7.0 (from retrying>=1.3.3->yoke)
  Using cached six-1.10.0.tar.gz
Collecting ruamel.ordereddict (from ruamel.yaml>=0.11.11->yoke)
  Using cached ruamel.ordereddict-0.4.9.tar.gz
Collecting futures<4.0.0,>=2.2.0 (from s3transfer<0.2.0,>=0.1.10->boto3>=1.4.2->yoke)
  Using cached futures-3.1.1.tar.gz
Building wheels for collected packages: yoke, boto3, botocore, Jinja2, jsonref, lambda-uploader, retrying, ruamel.yaml, jmespath, s3transfer, python-dateutil, docutils, MarkupSafe, virtualenv, six, ruamel.ordereddict, futures
  Running setup.py bdist_wheel for yoke ... error
  Complete output from command <snip>/venv/bin/python2.7 -u -c "import setuptools, tokenize;__file__='/private/var/folders/06/t0gs2dqj52x16pld4bn_4lk40v81z4/T/pip-build-m5GoeB/yoke/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /var/folders/06/t0gs2dqj52x16pld4bn_4lk40v81z4/T/tmpLBDVWcpip-wheel-:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib
  creating build/lib/yoke
  copying yoke/__init__.py -> build/lib/yoke
  copying yoke/config.py -> build/lib/yoke
  copying yoke/deploy.py -> build/lib/yoke
  copying yoke/shell.py -> build/lib/yoke
  copying yoke/templates.py -> build/lib/yoke
  copying yoke/utils.py -> build/lib/yoke
  installing to build/bdist.macosx-10.12-x86_64/wheel
  running install
  running install_lib
  creating build/bdist.macosx-10.12-x86_64
  creating build/bdist.macosx-10.12-x86_64/wheel
  creating build/bdist.macosx-10.12-x86_64/wheel/yoke
  copying build/lib/yoke/__init__.py -> build/bdist.macosx-10.12-x86_64/wheel/yoke
  copying build/lib/yoke/config.py -> build/bdist.macosx-10.12-x86_64/wheel/yoke
  copying build/lib/yoke/deploy.py -> build/bdist.macosx-10.12-x86_64/wheel/yoke
  copying build/lib/yoke/shell.py -> build/bdist.macosx-10.12-x86_64/wheel/yoke
  copying build/lib/yoke/templates.py -> build/bdist.macosx-10.12-x86_64/wheel/yoke
  copying build/lib/yoke/utils.py -> build/bdist.macosx-10.12-x86_64/wheel/yoke
  running install_egg_info
  running egg_info
  writing requirements to yoke.egg-info/requires.txt
  writing yoke.egg-info/PKG-INFO
  writing top-level names to yoke.egg-info/top_level.txt
  writing dependency_links to yoke.egg-info/dependency_links.txt
  writing entry points to yoke.egg-info/entry_points.txt
  reading manifest file 'yoke.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  writing manifest file 'yoke.egg-info/SOURCES.txt'
  Copying yoke.egg-info to build/bdist.macosx-10.12-x86_64/wheel/yoke-0.3.0-py2.7.egg-info
  running install_scripts
  error: [Errno 2] No such file or directory: 'LICENSE'

  ----------------------------------------
  Failed building wheel for yoke
  Running setup.py clean for yoke
  Running setup.py bdist_wheel for boto3 ... done
<snip>
Successfully built boto3 botocore Jinja2 jsonref lambda-uploader retrying ruamel.yaml jmespath s3transfer python-dateutil docutils MarkupSafe virtualenv six ruamel.ordereddict futures
Failed to build yoke
ERROR: Failed to build one or more wheels
```

**Comment by ryandub** [comment]
I haven't encountered this before. I guess LICENSE needs to be in the manifest?

**Comment by ryandub** [comment]
Fixed by #40
